src/seq_mv/csr_spgemm_device_symbl.h

```diff
@@ -389,8 +389,8 @@ hypre_spgemm_symbolic( hypre_DeviceItem             &item,
          }
          else
          {
-            failed = (char) group_reduce_sum<hypre_int, NUM_GROUPS_PER_BLOCK, GROUP_SIZE>(item,
```

Comment:
Why do we need to change to `HYPRE_Int`?

Reply:
@waynemitchell This may be fine for now.  The reason I don't like it is due to the fact that the reduction type is just a `char`. Casting it to `int` is an overkill already, I guess. But we can leave the change as is, since it only takes effect for `BigInt` build on GPUs, which I plan to work on later. Thank you for clarifying it for me.